### PR TITLE
MC-16115: added doc for list and get

### PR DESCRIPTION
**Pull request by jasondias9**
#### Related PRs
- [Core](https://github.com/cloudops/cloudmc-core/pull/1503/)

### CODE REVIEW

**Review by razine-bensari — APPROVED** [review]
Read it all, did not find any typos, looks good to me!!
🔥

File: source/includes/administration/_tax_providers.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
+### Tax providers
+
+A tax provider enables a reseller to map tax codes to monetization products. The tax code provider type will ensure that invoices include taxes based 
+on the billing details (i.e. billing address) provided by the reseller.
```
Comment (y0geshdev) [inline source/includes/administration/_tax_providers.md]: What does` tax code provider type` means?
The tax code type or the tax code provider?

Comment (jasondias9) [reply source/includes/administration/_tax_providers.md]: `tax code provider` thats a typo. Fixed here [`eaac51d` (#425)](https://github.com/cloudops/cloudmc-api-docs/pull/425/commits/eaac51d7c8225b1aa3cd4c18922f5eb6f208ae70)

File: source/includes/administration/_tax_providers.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+on the billing details (i.e. billing address) provided by the reseller. 
+
+<!-------------------- LIST TAX PROVIDERS -------------------->
+#### List tax providers
+
+`GET /tax_providers`
+
+Retrieves a list of tax providers configured for a reseller.
+
+> Note: You must have the Reseller billing permission on the target reseller. If the caller does not have the correct permissions or the reseller with the ID provided does not exist a 404 Not Found will be returned
```
Comment (y0geshdev) [inline source/includes/administration/_tax_providers.md]: nitpick if make sense -
```suggestion
> Note: You must have the Reseller billing permission on the target reseller. If the caller does not have the correct permissions or the reseller with the ID provided does not exist then a `404 Not Found` response will be returned.
```

File: source/includes/administration/_tax_providers.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+      }
+    }
+  ]
+}
+```
+
+Attributes | &nbsp;
+---- | -----------
+`id`<br/>*UUID* | The UUID of the tax provider.
+`reseller.id`<br/>*UUID* | UUID of the reseller to which the tax provider belongs. (AKA an organization id that is a reseller)
```
Comment (y0geshdev) [inline source/includes/administration/_tax_providers.md]:
```suggestion
`reseller.id`<br/>*UUID* | UUID of the reseller to which the tax provider belongs (AKA an organization id that is a reseller).
```

File: source/includes/administration/_tax_providers.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+`type`<br/>*string* | The type of tax provider. Possible values are: `AVALARA`.
+`state`<br/>*enum"  | The state of the tax provider. Possible values are: `CONFIGURED`, `CONFIGURING`, `ERROR`
```
Comment (y0geshdev) [inline source/includes/administration/_tax_providers.md]: missing period in the end - `.`

Comment (jasondias9) [reply source/includes/administration/_tax_providers.md]:
```suggestion
`state`<br/>*enum"  | The state of the tax provider. Possible values are: `CONFIGURED`, `CONFIGURING`, `ERROR`.
```

File: source/includes/administration/_tax_providers.md [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+
+
+<!-------------------- GET TAX PROVIDER -------------------->
+#### Get a tax provider
+
+`GET /tax_providers/:id`
+
+Retrieves a list of tax providers configured for a reseller.
+
+> Note: You must have the Reseller billing permission on the owner of the tax provider requested. If the caller does not have the correct permissions or the tax provider for the given ID does not exist a 404 Not Found will be returned
```
Comment (y0geshdev) [inline source/includes/administration/_tax_providers.md]: Should update this as well if we decide to change the first one - https://github.com/cloudops/cloudmc-api-docs/pull/425/files#r709416920

File: source/includes/administration/_tax_providers.md [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+      "password" "password"
+    }
+  }
+}
+```
+
+Attributes | &nbsp;
+---- | -----------
+`id`<br/>*UUID* | The UUID of the tax provider.
+`reseller.id`<br/>*UUID* | UUID of the reseller to which the tax provider belongs. (AKA an organization id that is a reseller)
```
Comment (y0geshdev) [inline source/includes/administration/_tax_providers.md]: same same

File: source/includes/administration/_tax_providers.md [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+`type`<br/>*string* | The type of tax provider. Possible values are: `AVALARA`.
+`state`<br/>*enum"  | The state of the tax provider. Possible values are: `CONFIGURED`, `CONFIGURING`, `ERROR`
```
Comment (y0geshdev) [inline source/includes/administration/_tax_providers.md]: same same same :P

**Review by y0geshdev — CHANGES_REQUESTED** [review]
Left some very minor suggestions. Not all need changes.

File: source/includes/administration/_tax_providers.md (hunk 5 above)
Comment (jasondias9) [reply source/includes/administration/_tax_providers.md]:
```suggestion
> Note: You must have the Reseller billing permission on the owner of the tax provider requested. If the caller does not have the correct permissions or the tax provider for the given ID does not exist, then a `404 Not Found` response will be returned.
```

File: source/includes/administration/_tax_providers.md (hunk 6 above)
Comment (jasondias9) [reply source/includes/administration/_tax_providers.md]:
```suggestion
`reseller.id`<br/>*UUID* | UUID of the reseller to which the tax provider belongs (AKA an organization id that is a reseller).
```

File: source/includes/administration/_tax_providers.md (hunk 7 above)
Comment (jasondias9) [reply source/includes/administration/_tax_providers.md]:
```suggestion
`state`<br/>*enum"  | The state of the tax provider. Possible values are: `CONFIGURED`, `CONFIGURING`, `ERROR`.
```

**Review by y0geshdev — APPROVED** [review]
Looks 🔥🔥. 🙏

**Pull request merged**